File: runtime/javascript/a+b.ts
import * as readline from "node:readline";

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.on("line", (input: string) => {
  const numbers = input.split(" ").map(Number);
  const sum = numbers[0] + numbers[1];
  console.log(sum);
  rl.close();
});

File: runtime/java/a+b.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        while (scanner.hasNextLine()) {
            String line = scanner.nextLine();
            String[] numbers = line.split(" ");
            int a = Integer.parseInt(numbers[0]);
            int b = Integer.parseInt(numbers[1]);
            int sum = a + b;
            System.out.println(sum);
        }
        scanner.close();
    }
}
